js/src/name-pubsub/cancel.js

```diff
@@ -63,24 +52,30 @@ module.exports = (createCommon, options) => {
 
     it('should cancel a subscription correctly returning true', function (done) {
       this.timeout(300 * 1000)
-      const ipnsPath = `/ipns/${nodeId}`
-
-      series([
-        (cb) => ipfs.name.pubsub.subs(cb),
-        (cb) => ipfs.name.publish(value, { resolve: false }, cb),
-        (cb) => ipfs.name.resolve(nodeId, cb),
-        (cb) => ipfs.name.pubsub.subs(cb),
-        (cb) => ipfs.name.pubsub.cancel(ipnsPath, cb),
-        (cb) => ipfs.name.pubsub.subs(cb)
-      ], (err, res) => {
+      const id = 'QmNP1ASen5ZREtiJTtVD3jhMKhoPb1zppET1tgpjHx2NGA'
+      const ipnsPath = `/ipns/${id}`
```

Comment:
Using a hard coded peer ID like this is fine as long as other developers don't come along and copy it and use it in their tests. Personally I'd generate one for the test.